portfolio/src/main/webapp/index.html

```diff
@@ -58,8 +58,8 @@ <h1>My Portfolio - Saakshi Saxena</h1>
             <br>
             <div id="contact-me">
                 <p style="font-size: larger;"><strong><em>Get in touch with me:</em></strong></p>
-                <a href="https://www.linkedin.com/in/saakshi-saxena-29797b174/" target="_blank"><img src="my-images/linkedin-pic.jpg" alt="Linkedin icon" style= "height:150px; width:150px;"/></a>
-                <a href="https://github.com/saakshisaxena" target="_blank"><img src="my-images/github-pic-light.png" alt="GitHub icon" style= "height:150px; width:150px;"/></a>
+                <a href="https://www.linkedin.com/in/saakshi-saxena-29797b174/" target="_blank"><img class="contact-me-icon" src="my-images/linkedin-pic.jpg" alt="Linkedin icon"/></a>
+                <a href="https://github.com/saakshisaxena" target="_blank"><img class="contact-me-icon" src="my-images/github-pic-light.png" alt="GitHub icon"/></a>
```

Comment:
Optional: Consider placing <img> tag on a new line to reduce the line length.

Reply:
Ok noted.